curriculum/challenges/english/25-front-end-development/lecture-css-specificity-the-cascade-algorithm-and-inheritance/672b8ea434ceac23cc90f337.md

```diff
@@ -86,22 +86,22 @@ This selector targets elements with a specific, unique identifier.
 
 ## --text--
 
-Which of the following selectors has a lower specificity than an ID selector?
+Which of the following selectors has a higher specificity than an ID selector?
```

Comment:
inline styles is not a selector. so selector should be removed from the question here.

```suggestion
Which of the following has a higher specificity than an ID selector?
```